php/php.editor/src/org/netbeans/modules/php/editor/actions/FixUsesPerformer.java

```diff
@@ -148,14 +146,24 @@ private void processSelections() {
             }
         }
         replaceUnimportedItems();
-        String insertString = createInsertString(useParts);
+        String insertString = createInsertString(useParts, shouldAppendLineAfter(startOffset));
         // avoid being recognized as a modified file
         if (insertString.isEmpty()) {
             StatusDisplayer.getDefault().setStatusText(Bundle.FixUsesPerformer_noChanges());
         } else {
             editList.replace(startOffset, 0, insertString, false, 0);
         }
     }
+    
+    private boolean shouldAppendLineAfter(int lineOffset)
+    {
+        try {
+            return LineDocumentUtils.getNextNonWhitespace(baseDocument, lineOffset, LineDocumentUtils.getLineEnd(baseDocument, lineOffset)) != -1;
+        }  catch (BadLocationException ex) {
+            Exceptions.printStackTrace(ex);
+        }
+        return false;
```

Comment:
would return in catch block read better?

Reply:
i am not a java developer, if it would be better i am willing to make changes, but just checked few files with similar logic and there is usually return outside